dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHV1IT.java

```diff
@@ -143,15 +247,12 @@ protected void assertTool(String tool, boolean isTool) {
      * This tests if the 4 workflows with a combination of different repositories and either same or matching workflow name
      * can be retrieved separately
      *
-     * @throws Exception
      */
     @Test
     public void toolsIdGet4Workflows() throws Exception {
         // Insert the 4 workflows into the database using migrations
         CommonTestUtilities.setupSamePathsTest(SUPPORT);
-        Response response2 = checkedResponse(basePath + "tools");
-        List<ToolV1> responseObject2 = response2.readEntity(new GenericType<List<ToolV1>>() {
-        });
```

Comment:
i think these 3 lines were meant to test if the /tools endpoint returned the correct structure

Reply:
Ah, I probably accidentally stripped it out via IDE refactoring. 
Adding back in and locking down with an assert